src/components/incentives/SimpleExternalIncentivesTooltip.tsx

```diff
@@ -0,0 +1,111 @@
+import { Trans } from '@lingui/macro';
+
+import { Link } from '../primitives/Link';
+import { TextWithTooltip } from '../TextWithTooltip';
+
+export const SuperFestTooltip = () => {
```

Comment:
Same here these can be extracted into a component i think all the logic and text is the same.